Project.toml

```diff
@@ -39,36 +74,5 @@ TriplotBase = "981d1d27-644d-49a2-9326-4793e63143c3"
 TriplotRecipes = "808ab39a-a642-4abf-81ff-4cb34ebbffa3"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
-[compat]
-CodeTracking = "1.0.5"
-ConstructionBase = "1.3"
-EllipsisNotation = "1.0"
-ForwardDiff = "0.10.18"
-GeometryBasics = "0.3, 0.4"
-HDF5 = "0.14, 0.15, 0.16"
-IfElse = "0.1"
-LinearMaps = "2.7, 3.0"
-LoopVectorization = "0.12.54"
-MPI = "0.19"
-MuladdMacro = "0.2.2"
-Octavian = "0.3.5"
-OffsetArrays = "1.3"
-P4est = "0.3.0"
-Polyester = "0.3.4, 0.5, 0.6"
-RecipesBase = "1.1"
-Reexport = "1.0"
-Requires = "1.1"
-SciMLBase = "1.21"
-Setfield = "0.8"
-StartUpDG = "0.12.5"
-Static = "0.3, 0.4, 0.5, 0.6"
-StaticArrays = "1.0"
-StrideArrays = "0.1.12"
-StructArrays = "0.6"
-SummationByPartsOperators = "0.5.10"
-TimerOutputs = "0.5"
-Triangulate = "2.0"
-TriplotBase = "0.1"
-TriplotRecipes = "0.1"
-UnPack = "1.0"
-julia = "1.7"
+[extras]
+CPUSummary = "2a0fbf3d-bb9c-48f3-b0a9-814d99fd7ab9"
```

Comment:
```suggestion
```
Do you use this anywhere? If not, we can remove it, I think

Reply:
Will do. I didn't realize it got added again 😠